### PR TITLE
Use system provided light font instead of Roboto-Light

### PR DESCRIPTION
**Pull request by nielsvanvelzen**
Uses the system provided sans-serif-light font (which by default is Roboto Light) instead of hardcoding our own (outdate) version of Roboto Light.

For most devices everything will look exactly the same, devices using a different system font will look a bit different.

When possible the font family declaration is done in XML. Files touched have their author comments removed and imports optimized.

### CODE REVIEW

File: app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java [hunk 1]
```diff
@@ -267,18 +266,12 @@ public void onActivityCreated(Bundle savedInstanceState) {
         setupPopupAnimations();
 
         //live guide
-        Typeface roboto = TvApp.getApplication().getDefaultFont();
         mDisplayDate = mActivity.findViewById(R.id.displayDate);
         mGuideTitle = mActivity.findViewById(R.id.guideTitle);
-        mGuideTitle.setTypeface(roboto);
         mGuideCurrentTitle = mActivity.findViewById(R.id.guideCurrentTitle);
-        mGuideCurrentTitle.setTypeface(roboto);
```
Comment (AndreasGB) [inline app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java]: Missing xml adjustment?

File: app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java [hunk 2]
```diff
@@ -267,18 +266,12 @@ public void onActivityCreated(Bundle savedInstanceState) {
...
         mSummary = mActivity.findViewById(R.id.summary);
-        mSummary.setTypeface(roboto);
```
Comment (AndreasGB) [inline app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java]: Missing xml adjustment?

File: app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java (hunk 1 above)
Comment (nielsvanvelzen) [reply app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java]: We search for the view in the `mActivity` which is `PlaybackOverlayActivity`. This inflates the `activity_playback_overlay` layout. This uses the `CustomPlaybackOverlayFragment` fragment (at this point I'm wondering why it doesn't use `this` instead of `mActivity`) which inflates `vlc_player_interface`.

I can't find where it exactly initializes the XML but the **only** file containing an id `guideCurrentTitle` was `overlay_tv_guide` so I adjusted the XML in that file.


TL;DR:: everything should be set for those 2

File: app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java (hunk 2 above)
Comment (nielsvanvelzen) [reply app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java]: See other comment

File: app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java (hunk 1 above)
Comment (AndreasGB) [reply app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java]: I don't see it [there](https://github.com/jellyfin/jellyfin-androidtv/blob/614865563c72b66e7ab9175d85e292af61e9ae44/app/src/main/res/layout/overlay_tv_guide.xml#L196), or am I misunderstanding this?

Comment (nielsvanvelzen) [reply app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java]: You're right, I somehow missed that 2 times lol

Comment (AndreasGB) [reply app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java]: Happens :D
